.nvmrc

```diff
@@ -0,0 +1 @@
+10.14.1
```

Comment:
```suggestion
10.24.1
```

Reply:
Happy to do that, IIRC it was because the deployment was on 10.14.1, I was looking over someone's shoulder on a screenshare. But looking again, I cannot find that. I'll set it to current 10.x.x latest before merge.

Reply:
Right that makes sense, our beanstalk setup could well be running that indeed.
Hopefully we move to 16 soon enough :pray: